secretmanager/api/QuickStart/README.md

```diff
@@ -0,0 +1,41 @@
+# .NET Secret Manager Sample
+
+A sample that demonstrates how to call the
+[Secret Manager API](https://cloud.google.com/secret-manager/docs) from C#.
+
+This sample requires [.NET Core 2.1](
+    https://www.microsoft.com/net/core) or later.
+
+## Build and Run
+
+1.  **Follow the set-up instructions in [the documentation](https://cloud.google.com/dotnet/docs/setup).**
+
+1.  Enable APIs for your project.
+    [Click here](https://console.cloud.google.com/flows/enableapi?apiid=secretmanager.googleapis.com&showconfirmation=true)
+    to visit Cloud Platform Console and enable the Secret Manager API.
+
+1.  Edit [QuickStart.cs](QuickStart/QuickStart.cs), and replace
```

Comment:
Good catch. I'm now guarding that with a conditional. It's not just project, it's also secret id. The tests randomly generate a UUID (because secret names have to be unique) and also cleans them up afterward. The only other option I can think of is an environment variable, but I think this is a lot cleaner (and what we did in the samples of the other languages).